File: Python/programaMultiplo.py
# Elaborar un programa que verifique que, si un número es par se lo multiplique por 3 y se lo eleve al cuadrado.

ingreseNumero = int(input("Ingresa un número: "))

if ingreseNumero % 2 == 0:
    primerSolucion = ingreseNumero * 3
    segundaSolucion = pow(primerSolucion, 2)
    print(f"El resultado es: {segundaSolucion}")

File: C++/programaMultiplo.cpp
/*Elaborar un programa que verifique que, si un número es par se lo multiplique por 3 y se lo eleve al cuadrado.*/

#include <iostream>
using namespace std;

int main()
{
    int ingresarNumero, primerSolucion, segundaSolucion;
    int numero = 3;

    cout << "Ingresa un número: ";
    cin >> ingresarNumero;

    if (ingresarNumero % 2 == 0)
    {
        primerSolucion = ingresarNumero * numero;
        segundaSolucion = pow(primerSolucion, 2);
        cout << "La solución es: " << segundaSolucion;
    }
    else
    {
        cout << "El número " << ingresarNumero << " no es PAR.";
    }

    return 0;
}
